app_dart/lib/src/request_handlers/github_rate_limit_status.dart

```diff
@@ -38,9 +38,11 @@ class GithubRateLimitStatus extends RequestHandler<Body> {
 
     final int remainingQuota = quotaUsage['remaining'] as int;
     const int quotaLimitSLO = 7500; // Half of the quota limit 15000.
-    if(remainingQuota < quotaLimitSLO) {
-      log.warning('Remaining GitHub quota is $remainingQuota, which is less than $quotaLimitSLO (half of the quotaLimit 15000).');
+    if (remainingQuota < quotaLimitSLO) {
+      log.warning(
+          'Remaining GitHub quota is $remainingQuota, which is less than $quotaLimitSLO (half of the quotaLimit 15000).');
```

Comment:
Remove the 15000, it will make this more future proof.

Reply:
I thought the quota was 16000?

Reply:
use variable instead.